js/modules/loki_snode_api.js

```diff
@@ -490,6 +508,7 @@ class LokiSnodeAPI {
       throw new window.textsecure.SeedNodeError('Failed to contact seed node');
     }
     log.info('loki_snodes:::refreshRandomPoolPromise - RESOLVED');
+    this.refreshRandomPoolPromise = false; // clear any lock
```

Comment:
I would rather set it to `null` or `undefined` than `false`

Reply:
I try to avoid null generally due stuff I've read like https://medium.com/@hbarcelos/why-i-banned-null-from-my-js-code-and-why-you-should-too-13df90323cfa